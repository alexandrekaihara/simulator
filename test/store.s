LOAD MINUSTHREE
STORE AUX
LOAD AUX
SUB MINUSTHREE
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO:
OUTPUT ONE
STOP
MINUSTHREE: CONST -3
AUX: SPACE
ONE: CONST 1
MINUSONE: CONST -1