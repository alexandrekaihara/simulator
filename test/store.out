10 16 11 17 10 17 2 16 8 13 13 19 14 13 18 14 -3 0 1 -1